portal-api/.eslintrc.js

```diff
@@ -41,6 +41,7 @@ const baseRules = {
       consistent: true,
     },
   ],
+  'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['draft', 'req'] }],
```

Comment:
in order to use immer this is needed

Reply:
I agree here, although I know its been taken out before -- Once we have sign off that this is good, could we keep the other `.eslintrc`s updated with this change please.

The alternative for this is a lot of eslint ignores which are not good for the code health